pkg/doc/presexch/definition.go

```diff
@@ -888,6 +962,24 @@ func (a byID) Len() int           { return len(a) }
 func (a byID) Less(i, j int) bool { return a[i].ID < a[j].ID }
 func (a byID) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 
+func filterFormat(format *Format, credentials []*verifiable.Credential) []*verifiable.Credential {
+	var result []*verifiable.Credential
+
+	if format.LdpVP == nil {
+		return result
+	}
+
+	for _, credential := range credentials {
+		for _, proofType := range format.LdpVP.ProofType {
+			if hasProofWithType(credential, proofType) {
+				result = append(result, credential)
+			}
+		}
+	}
```

Comment:
how we are handling JWT VC here?
Do we have a follow up task on that?

Reply:
We can add JWT VC to the list of supported formats here.